gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/PipelineTemplatesController.java

```diff
@@ -123,6 +123,24 @@ public Map update(@PathVariable String id, @RequestBody Map<String, Object> pipe
     return taskService.create(operation);
   }
 
+  @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
+  @ResponseStatus(value = HttpStatus.ACCEPTED)
+  public Map delete(@PathVariable String id) {
+    List<Map<String, Object>> jobs = new ArrayList<>();
+    Map<String, Object> job = new HashMap<>();
+    job.put("type", "deletePipelineTemplate");
+    job.put("pipelineTemplateId", id);
+    jobs.add(job);
+
+    Map<String, Object> operation = new HashMap<>();
+    operation.put("description", "Delete pipeline template '" + id + "'");
+    // TODO: DEFAULT_APPLICATION doesn't feel right, but the user shouldn't have to send anything more than the template's id.
+    operation.put("application", DEFAULT_APPLICATION);
```

Comment:
I agree - I'm not stoked at all about `DEFAULT_APPLICATION`. Perhaps if we made it an optional query param? We don't need to figure this out today, though.